crowdsec-docs/unversioned/user_guides/log_centralization.md

```diff
@@ -0,0 +1,248 @@
+---
+id: log_centralization
+title: Log Centralization
+sidebar_position: 10
+---
+
```

Comment:
Maybe explain the "why" ie. we want to centralize logs so we can run only one LP/SE